### PR TITLE
Pagination nav updates

### PR DESCRIPTION
**Pull request by jweakley**

### CODE REVIEW

File: addon/templates/components/freestyle/bootstrap-pagination-nav.hbs [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+      label='Page Size Option'
+      value=pageSize}}
+    {{bootstrap-checkbox
+      isChecked=showSinglePage
+      label='Show Single Page Option'
+      onclick=(action (mut showSinglePage))}}
+  {{/freestyle-annotation}}
   {{#freestyle-usage "pagination" title="Pagination"}}
     {{bootstrap-pagination-nav
+      showSinglePage=showSinglePage
```
Comment (Samsinite) [inline addon/templates/components/freestyle/bootstrap-pagination-nav.hbs]: I feel like this whole thing could have been solved with:
```
{{#if (gt totalPages 1)}}
  {{bootstrap-pagination-nav
    pageNumber=pageNumber
    pageSize=pagesize
    totalPages=totalPages
    moveToPage=(action (mut pageNumber))
  }}
{{/if}}
```

And that this is just adding code to support something that can already be done. Thoughts?

Comment (jweakley) [reply addon/templates/components/freestyle/bootstrap-pagination-nav.hbs]: Updating it to remove option. I am adding this snippet for this in the readme.

**Comment by jweakley** [comment]
@Samsinite this is ready for review and merge.

File: addon/components/freestyle/bootstrap-pagination-nav.js [hunk 2]
```diff
@@ -3,9 +3,24 @@ import layout from '../../templates/components/freestyle/bootstrap-pagination-na
...
-    return (100 / this.get('pageSize'));
+
+  totalPages: Ember.computed('pageSize', 'numberOfElements', function() {
+    if(this.get('pageSize') && this.get('pageSize') > 0) {
+      return Math.ceil(this.get('numberOfElements') / this.get('pageSize'));
+    } else {
+      return this.get('numberOfElements');
+    }
+  }),
+  elements: Ember.computed('pageSize', 'numberOfElements', 'pageNumber', function() {
```
Comment (Samsinite) [inline addon/components/freestyle/bootstrap-pagination-nav.js]: I don't see `numberOfElements` used here, should be able to remove the watch on that property.

**Pull request merged**